esmvaltool/cmorizers/data/formatters/datasets/nsidc_g02202_sh.py

```diff
@@ -0,0 +1,182 @@
+    cubes = iris.load(filepath, raw_var, _prom_dim_coord)
+    iris.util.equalise_attributes(cubes)
+
+    cube = cubes.concatenate_cube()
+    iris.util.promote_aux_coord_to_dim_coord(cube, 'projection_y_coordinate')
+    iris.util.promote_aux_coord_to_dim_coord(cube, 'projection_x_coordinate')
+    cube.coord('projection_y_coordinate').rename('y')
+    cube.coord('projection_x_coordinate').rename('x')
+
+    cube.add_aux_coord(latlon[0], (1, 2))
+    cube.add_aux_coord(latlon[1], (1, 2))
+    # add coord typesi
+    area_type = AuxCoord([1.0], standard_name='area_type', var_name='type',
+                         long_name='Sea Ice area type')
+    cube.add_aux_coord(area_type)
+
+    # cube.convert_units(cmor_info.units)
+    cube.units = '%'
+    cube.data[cube.data > 100] = np.nan
+    cube = cube * 100
+
+    # utils.fix_coords(cube) #latlon multidimensional
+    utils.fix_var_metadata(cube, cmor_info)
+    utils.set_global_atts(cube, attrs)
+
+    utils.save_variable(cube,
+                        var,
+                        out_dir,
+                        attrs,
+                        unlimited_dimensions=['time'])
+
+    return cube
+
+
+def _create_areacello(cfg, in_dir, sample_cube, glob_attrs, out_dir):
+    if not cfg['custom'].get('create_areacello', False):
+        return
+    var_info = cfg['cmor_table'].get_variable('Ofx', 'areacello')
+    glob_attrs['mip'] = 'Ofx'
+    lat_coord = sample_cube.coord('latitude')
+
+    area_file = os.path.join(in_dir, cfg['custom']['area_file'])
+    with open(area_file, 'rb') as datfile:
+        areasdmnd = np.fromfile(datfile,
+                                dtype=np.int32).reshape(lat_coord.shape)
+
+    # Divide by 1000 to get km2 then multiply by 1e6 to m2 ...*1000
+    ardata = areasdmnd * 1000
+
+    cube = iris.cube.Cube(ardata,
+                          standard_name=var_info.standard_name,
+                          long_name=var_info.long_name,
+                          var_name=var_info.short_name,
+                          units='m2',
+                          dim_coords_and_dims=[(sample_cube.coord('y'), 0),
+                                               (sample_cube.coord('x'), 1)])
+    cube.add_aux_coord(lat_coord, (0, 1))
+    cube.add_aux_coord(sample_cube.coord('longitude'), (0, 1))
+    utils.fix_var_metadata(cube, var_info)
+    utils.set_global_atts(cube, glob_attrs)
+    utils.save_variable(cube, var_info.short_name, out_dir, glob_attrs,
+                        zlib=True)
+
+
+def cmorization(in_dir, out_dir, cfg, cfg_user, start_date, end_date):
+    """Cmorization func call."""
+    glob_attrs = cfg['attributes']
+    cmor_table = cfg['cmor_table']
+
+    # get aux nc file
+    cubesaux = iris.load(os.path.join(in_dir, 'G02202-cdr-ancillary-sh.nc'))
+    lat_coord = _create_coord(cubesaux, 'lat', 'latitude')
+    lon_coord = _create_coord(cubesaux, 'lon', 'longitude')
+    year = 1978
+    # split by year..
+    sample_cube = None
+    while year <= 2020:
```

Comment:
This should probably be 2022 to process all data downloaded and specified in `recipe_check_obs.yml`.

```suggestion
    while year <= 2022:
```